projects/RabbitMQ.Client/client/impl/Frame.cs

```diff
@@ -136,25 +139,17 @@ internal static class Heartbeat
             /// <summary>
             /// Compiler trick to directly refer to static data in the assembly, see here: https://github.com/dotnet/roslyn/pull/24621
             /// </summary>
-            private static ReadOnlySpan<byte> Payload => new byte[]
+            internal static ReadOnlySpan<byte> Payload => new byte[]
```

Comment:
changing this most likely does fail the compiler trick (see comment)

Reply:
It shouldn't. The examples I saw used both private and public modifiers.

Reply:
Works the same :)
![image](https://user-images.githubusercontent.com/661658/96011339-54df3500-0e32-11eb-9987-87a971fc9390.png)